tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -1211,8 +1209,6 @@ silent_mode_error_trap() {
 }
 
 main() {
-	git config --global --add safe.directory ${repo_root_dir}
```

Comment:
There are other places that do that, e.g. `get_last_modification()` at https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/scripts/lib.sh#L111 that has a lot of callers in this script.

Please address the removal of this workaround globally in a distinct PR.

Reply:
Right, AFAIR we have them also  sprinkled in the gh actions as well.